spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/audit/service/DefaultAuditRecordService.java

```diff
@@ -111,6 +111,6 @@ else if (actions != null && operations != null) {
 
 	@Override
 	public AuditRecord findOne(Long id) {
-		return this.auditRecordRepository.findOne(id);
+		return this.auditRecordRepository.findById(id).orElse(null);
```

Comment:
Isn't this already returning null based on standard spring data JPA conventions?

Reply:
In spring data, `findById` now returns `Optional`, so `orElse` either returns its value or null if there's no value. We need to revisit if we want to rewrite whole stack to use optionals as well but as of now we rely on plain values which can be null. Plain `Optional.get()` throws error if value is null.